src/shared/services/transform/index.test.js

```diff
@@ -0,0 +1,17 @@
+import { formatWeekOrWorkdays } from '.';
+
+describe('transform service', () => {
```

Comment:
Could you add tests for `getHandlingTimesBySlugFromSubcategories`?

Reply:
Done, method was covered by tests from the components and this test is not required if we would follow the `testing-library best practices`. That said it crossed my mind to explicitly add this test since we are inconsistent by applying those best practices anyway, test has been added ;)